samples/mIRC Script/sample.mrc

```diff
@@ -0,0 +1,33 @@
+; Simple !hello response script
+
+on *:text:!hello:#: { msg # Hello there! I am a simple response script. }

```

Comment:
We prefer to avoid hello world samples because they are then used to train the Bayesian classifier. Any chance you could find a different sample from a "real" application?

Reply:
Sure, I have a few old scripts that don't really work well anymore (they scraped websites that have changed but the script does execute and run properly). Would those work?

Reply:
There we go. 

The `torncity-*` scripts are my own work. MIT license.

The AutoHostmeBot.mrc came from [here](https://github.com/BrandonDocDC/AutoHostmeBot). GPL 3.0 license.

Reply:
I'd rather we remove the hello world sample. It's only going to skew the classifier's results. If it's needed for tests, it can be moved to `tests/fixtures/`.

Reply:
Ah, I misread - removed it. Sorry about that.

Reply:
> Oh, right. I forgot that. It will show up in a default grey color. We should really avoid that. Could you try colors a bit more? Even a random color would probably be better.

Got one - the first one I've been able to come across that was free after a few hours of testing. Not a very pretty one but... that's probably why it was free. 

I propose to add it to [this](https://github.com/github/linguist/issues/4506#issue-436984670) as well - logo [here](https://www.mirc.com/images/indexlogorb.gif).

Thanks :)